site/docs/v1/tech/shared/_deleting-your-account.adoc

```diff
@@ -0,0 +1,25 @@
+To delete your account, take the following steps, in this order:
+
+1. https://account.fusionauth.io/[Log into your account].
+2. Delete all deployments.
+3. Switch your account to the community plan.
+4. Delete your company.
+
+[NOTE.note]
+====
+FusionAuth requires that customers delete their own deployments using the account portal.
+
+FusionAuth is **not able** to cancel accounts with deployments that could potentially contain critical user data.
+====
+
+To remove your deployments, please https://account.fusionauth.io/account/deployment/[log into your account and navigate to your deployments]. Once you see your deployment, you can click the [uielement]#Action# button and select [uielement]#Delete#. After a deployment is destroyed, you won't be charged going forward. Any previous usage outside of a free trial will be billed.
+
+Then, https://account.fusionauth.io/account/plan/[switch your plan to to the community plan]. After the current billing cycle is done, you won't be charged. (Plans run for a month.)
+
+After deleting your deployments and changing your plan, your account will not be charged. It is inactive and can be deleted if you desire.
```

Comment:
> your account will not be charged.

After deleting your deployments and changing your plan, no additional charges will be generated by the company you have created in FusionAuth Cloud.

Reply:
It is perfectly acceptable to leave your account in this condition (especially in the case where you may be doing future work with FusionAuth).  However, if data practices or company policies mandate -- you may also remove your Account from FusionAuth Cloud.  Removing a company and its data is not reversible.